js/api/firebaseHelper.js

```diff
@@ -89,7 +87,64 @@ export function writeNewUserTests(
       testStartTime: testStartTime,
       testScore: testScore
     })
-    .then(function () {
+    .then(function() {
       console.log("User location written!");
     });
-}
\ No newline at end of file
+}
+// PROFILE CREATION
+export function writeNewUserBio(userID, bio) {
+  database
+    .collection("users")
+    .doc(userID)
+    .set(
+      {
+        bio: bio
+      },
+      { merge: true }
```

Comment:
Please be consistent with code formatting. It is just a good practice. Thank you!